src/mono/wasm/debugger/DebuggerTestSuite/Support.cs

```diff
@@ -301,54 +302,31 @@ internal void CheckNumber<T>(JToken locals, string name, T value)
                 var val = l["value"];
                 Assert.Equal("number", val["type"]?.Value<string>());
                 Assert.Equal(value, val["value"].Value<T>());
+                Assert.Equal(value.ToString(), val["description"].Value<T>().ToString());
                 return;
             }
             Assert.True(false, $"Could not find variable '{name}'");
         }
 
         internal void CheckString(JToken locals, string name, string value)
         {
-            foreach (var l in locals)
-            {
-                if (name != l["name"]?.Value<string>())
-                    continue;
-                var val = l["value"];
-                if (value == null)
-                {
-                    Assert.Equal("object", val["type"]?.Value<string>());
-                    Assert.Equal("null", val["subtype"]?.Value<string>());
-                }
-                else
-                {
-                    Assert.Equal("string", val["type"]?.Value<string>());
-                    Assert.Equal(value, val["value"]?.Value<string>());
-                }
-                return;
-            }
-            Assert.True(false, $"Could not find variable '{name}'");
+            var l = GetAndAssertObjectWithName(locals, name);
+            CheckValue(l["value"], TString(value), name).Wait();
```

Comment:
This appears to break things because TString assumes description is set.

Reply:
Break where?

Reply:
JSExceptionTestAll and ExceptionTestUncaught

Reply:
```
  Error Message:
   System.AggregateException : One or more errors occurred. (One or more errors occurred. ([message] Could not find value field named description
Expected: True
Actual:   False))
---- System.AggregateException : One or more errors occurred. ([message] Could not find value field named description
Expected: True
Actual:   False)
-------- [message] Could not find value field named description
Expected: True
Actual:   False
  Stack Trace:
```

Reply:
I removed the description check, we should double check but I don't think the frontends actually use the decscription field on string at all.